Composite/Plugins/Functions/WidgetFunctionProviders/StandardWidgetFunctionProvider/DataReference/HomePageSelectorWidgetFunction.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections;
+using System.Xml.Linq;
+using Composite.Core.WebClient.Renderings.Page;
+using Composite.Data;
+using Composite.Data.Types;
+using Composite.Functions;
+using Composite.Plugins.Functions.WidgetFunctionProviders.StandardWidgetFunctionProvider.Foundation;
+
+namespace Composite.Plugins.Functions.WidgetFunctionProviders.StandardWidgetFunctionProvider.DataReference
+{
+    /// <summary>    
+    /// </summary>
+    /// <exclude />
+    [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
+    public sealed class HomePageSelectorWidgetFunction : CompositeWidgetFunctionBase
+    {
+        /// <exclude />
+        public HomePageSelectorWidgetFunction(EntityTokenFactory entityTokenFactory)
+            : base(CompositeName, typeof(DataReference<IPage>), entityTokenFactory)
+        {
+        }
+
+        private const string CompositeName = CommonNamespace + ".DataReference" + ".HomePageSelector";
```

Comment:
HomePageSelector is defaultwidget now, but default widget should be PageSelector (Or optionalPage selector)

Reply:
should be changed

Reply:
![1](https://user-images.githubusercontent.com/14817424/71406557-3f99df80-2641-11ea-93b5-302969cc8ee5.gif)

Reply:
Did you validate? For some reason, I still have it default.
![image](https://user-images.githubusercontent.com/14817424/71406154-06ad3b00-2640-11ea-9ed2-c96cadc20ded.png)